### PR TITLE
Fix animation refs - firelion_front, firelion_right, snake_right, icelance, shot and others

### PR DESCRIPTION
**Pull request by AndyMender**
Minor fixes for: https://github.com/Tuxemon/Tuxemon/issues/1185

- firelion_right is 64x61 and will be pulled from the wiki in a follow-up to: https://github.com/Tuxemon/Tuxemon/pull/1306

- shot/watershot has currently no author so isn't added to the repository

### CODE REVIEW

File: mods/tuxemon/db/technique/flamethrower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "tech_id": 73,
   "accuracy": 0.8,
-  "animation": "fire lion right",
+  "animation": "firelion_right",
```
Comment (AndyMender) [inline mods/tuxemon/db/technique/flamethrower.json:4]: From what I saw, there is more slip-ups like this (spaces in slugs, wrong names, etc.). Should I clean these up in this PR or are we going to tackle them later once we have a rundown on the missing credits/attributions and non-64x64 anims?

Comment (kerizane) [reply mods/tuxemon/db/technique/flamethrower.json:4]: Fix as many as you can

Comment (AndyMender) [reply mods/tuxemon/db/technique/flamethrower.json:4]: Done!

**Review by kerizane — APPROVED** [review]

**Comment by kerizane** [comment]
Great, thanks!

**Pull request merged**